syntax = "proto3";

package framework.protos;

message Node {
  string node_id = 1;
}
